presets/4.3/vtx/Japanese_Bandplans.txt

```diff
@@ -0,0 +1,41 @@
+#$ TITLE: VTX Japanese Bandplans
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, Japan, Japanese, JP, JA, Bandplans
+#$ AUTHOR: ToshihiroMakuuchi
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: This VTX preset conforms to the Japanese band plans.
+#$ DESCRIPTION: However, it is not a perfect fit for all VTXs. Please use it only as a reference.
+#$ DESCRIPTION:
+#$ DESCRIPTION: -- Amateur Radio License in Japan --
+#$ DESCRIPTION: Radio communications most commonly available include amateur, civil, and personal radio communications.
+#$ DESCRIPTION: Amateur radio communications are authorized only for those who pass a national license examination or who complete JARD & other official training course and acquire the prescribed radio operator certificate.
```

Comment:
Could list here the allowed channels for the users that don't want to check the raw CLI text.

Reply:
I understand. I will add the permitted channels.
In Japan, seven frequencies are theoretically available, but only three can be used simultaneously due to radio interference. I will add that information as well.